.circleci/config.yml

```diff
@@ -1,54 +1,61 @@
 version: 2
 jobs:
-  build:
+  build-project:
     docker:
-      - image: checkstyle/sonarqube-maven-git:7.7-community-8c1737e
+      - image: cimg/openjdk:11.0.16
 
+    steps:
+      - checkout
+      - run: mvn package -Pno-validations
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - project
```

Comment:
We do not need composed images, since we can build jar and generate class files in previous step, and persist the project directory for sonar execution job.